op-batcher/batcher/config.go

```diff
@@ -194,4 +194,11 @@ func NewConfig(ctx *cli.Context) *CLIConfig {
 		RPC:                          oprpc.ReadCLIConfig(ctx),
 		AltDA:                        altda.ReadCLIConfig(ctx),
 	}
+
+	// altDA specific settings
+	if cfg.AltDA.Enabled {
+		derive.SetMaxFrameLenForAllDA(altda.MaxInputSize)
```

Comment:
As is this seems like it will hard code to `300_000_000`?

Reply:
MaxL1TxnSize is a Batcher config, whereas op-node is the one we need to change for block derivation. we could even remove this section, I believe, since batcher isn't responsible for block derivation. I forgot to push changes to op-node config yesterday. see latest changes

We set `altda.MaxInputSize` to  a large num so it can work with future altDA optimizations.  `MaxL1TxnSize ` is what really controls how large a batch can get.  plus, op-node config doesn't have access to batcher's `MaxL1TxnSize` config. 

in theory,  the correct thing to do is to derive `MaxL1TxnSize` (misnomer) from `altda.MaxInputSize` (shouldn't be a constant), but that requires more changes than we'd like to make

Reply:
ah I see, thanks for explaining!